husc/screens/cellomics.py

```diff
@@ -0,0 +1,263 @@
+"""Feature computations and other functions for Cellomics screens.
+"""
+
+import os
+import collections as coll
+from skimage import io
+import numpy as np
+
+
+def listdir_fullpath(dir):
+    """Extended os.listdir that lists the full directory instead of just the
+    filename.
+
+    Parameters
+    ----------
+    dir : string
+        The query directory.
+
+    Returns
+    -------
+    path : list of string
+        A list of files in the query directory with their full path.
+    """
+    path = [os.path.join(dir, fn) for fn in os.listdir(dir)]
+    return path
+
+
+def batch_snail_stitch(dict, dir):
+    """Run snail stitch over a dictionary of filenames and output to dir.
+
+    Parameters
+    ----------
+    dict : dict
+        A dictionary of (plate, well) co-ordinates mapped to filenames.
+        Dictionary built with ``make_wellchannel2file_dict`` function.
+    dir : string
+        The directory to output the files to.
+    """
+    for key, value in dict.iteritems():
```

Comment:
key, value can be more descriptive, and you're not even using the keys, so:

for filename in key2file.values():